hebiros_description/urdf/common_properties.xacro

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" ?>
+
+<robot name="xacro_properties" xmlns:xacro="http://www.ros.org/wiki/xacro">
```

Comment:
This file seems like overkill, and weirdly duplicitous of the existing `x_series_common`.  I think the only lines you use are "light_black" in here; why not just add this one material to the other common file?

Overall, I want to avoid this type of divergence as much as possible -- this is the type of clutter we should be cleaning up in our repo.